src/Regulator.h

```diff
@@ -112,12 +112,15 @@ class StdDev
     double lastMean;
     double lastMin;
     double lastMax;
+    int plcUnderruns;
+    int lastPlcUnderruns;
     double lastStdDev;
     double longTermStdDev;
     double longTermStdDevAcc;
+    double longTermMax;
+    double longTermMaxAcc;
+    double lastTime;
     int longTermCnt;
-    int plcUnderruns;
-    int lastPlcUnderruns;
```

Comment:
It looks like a lot of these could be private.

I would also like to see that we use mCamelCase for members everywhere. But this can be done automatically with clang-tidy. So don't start to change this manually.